resources/js/snippets.js

```diff
@@ -15,5 +15,6 @@ var snippets = {
     "Ruby" : "ruby.md",
     "Typescript" : "typescript.md",
     "YAML" : "yaml.md",
-    "Lua" : "lua.md"
+    "Lua" : "lua.md",
+    "Java" : "java.md"
```

Comment:
Put `"Java" : "java.md"` before `"JavaScript" : "javascript.md"`, so  it will  have chronological order.